packages/fast-components-react-base/src/listbox-item/listbox-item.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import { ListboxItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    ListboxItemHandledProps,
+    ListboxItemProps,
+    ListboxItemUnhandledProps,
+} from "./listbox-item.props";
+import {
+    ListboxContext,
+    ListboxContextType,
+    ListboxItemData,
+} from "../listbox/listbox-context";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+
+class ListboxItem extends Foundation<
+    ListboxItemHandledProps,
+    ListboxItemUnhandledProps,
+    {}
+> {
+    public static displayName: string = "ListboxItem";
+
+    public static contextType: React.Context<ListboxContextType> = ListboxContext;
+
+    public static defaultProps: Partial<ListboxItemProps> = {
+        disabled: false,
+    };
+
+    protected handledProps: HandledProps<ListboxItemHandledProps> = {
+        disabled: void 0,
+        displayString: void 0,
+        managedClasses: void 0,
+        id: void 0,
+        onInvoke: void 0,
+        value: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        let isSelected: boolean = false;
+        if ((this.context as ListboxContextType).listboxSelectedItems !== undefined) {
+            isSelected =
+                (this.context as ListboxContextType).listboxSelectedItems.filter(
+                    (item: ListboxItemData) => {
+                        return item.id === this.props.id;
+                    }
+                ).length === 1;
+        }
```

Comment:
nit/suggestion - it may be easier to grok this if we extract this to a function and just assign it to the aria-selected value as that is the only place this gets used. As it stands now, I expected the above selection logic to be used in multiple places.

Reply:
Yep, cleaner